### PR TITLE
arista-switch-pack

### PR DESCRIPTION
**Pull request by cweltPA**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-6624

## Description
A new XSIAM pack for Arista EOS Switches.

### CODE REVIEW

File: Packs/AristaSwitch/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+   Password:
+   switch# configure
+   switch(config)# logging host 192.168.0.10 514
+   switch(config)# write
+   switch(config)# exit
+   switch# exit
+```
+
+Remarks: 
+- By default, the "_**logging host**_" command described above configures the Syslog forwarding over UDP. If you wish to forward the event logs via a secure channel over TCP, check out the documentation in the following links:
```
Comment (ShirleyDenkberg) [inline Packs/AristaSwitch/README.md:33]:
```suggestion
- By default, the _**logging host**_ command described above configures the Syslog forwarding over UDP. If you wish to forward the event logs via a secure channel over TCP, refer to the documentation in the following links:
```

File: Packs/AristaSwitch/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  - [Syslog with TLS Support](https://www.arista.com/en/um-eos/eos-control-plane-security#xx1117976). 
+  - [Logging - Basic Syslog and Beyond](https://arista.my.site.com/AristaCommunity/s/article/logging-basic-syslog-and-beyond) (see the "_Secure Syslog Transmission_" section).
```
Comment (ShirleyDenkberg) [inline Packs/AristaSwitch/README.md:35]:
```suggestion
  - [Logging - Basic Syslog and Beyond](https://arista.my.site.com/AristaCommunity/s/article/logging-basic-syslog-and-beyond) (see the _Secure Syslog Transmission_ section).
```

File: Packs/AristaSwitch/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+- You may wish to customize the logging level to filter events from a certain level and/or facility. See [_Understanding Logging Levels_](https://arista.my.site.com/AristaCommunity/s/article/understanding-logging-levels) for additional details.
+- By default, the timestamps in the generated event logs are specified in the traditional [RFC3164](https://www.ietf.org/rfc/rfc3164.txt) syslog format, which does not include a year and a timezone. It is recommended you override this default setting and configure the switch to forward the Syslog messages in [RFC5424](https://datatracker.ietf.org/doc/html/rfc5424) format, which specifies a high-resolution [RFC3339](https://datatracker.ietf.org/doc/html/rfc3339) timestamp which does include a year and a timezone. This configuration could be done from the _Global Configuration_ command mode via the "_**logging format**_" command. See [Syslog Logging Format](https://www.arista.com/en/um-eos/eos-switch-administration-commands#xx1268462) for additional details.
```
Comment (ShirleyDenkberg) [inline Packs/AristaSwitch/README.md:37]:
```suggestion
- By default, the timestamps in the generated event logs are specified in the traditional [RFC3164](https://www.ietf.org/rfc/rfc3164.txt) syslog format, which does not include a year and a timezone. It is recommended you override this default setting and configure the switch to forward the Syslog messages in [RFC5424](https://datatracker.ietf.org/doc/html/rfc5424) format, which specifies a high-resolution [RFC3339](https://datatracker.ietf.org/doc/html/rfc3339) timestamp which does include a year and a timezone. This configuration could be done from the _Global Configuration_ command mode via the _**logging format**_ command. See [Syslog Logging Format](https://www.arista.com/en/um-eos/eos-switch-administration-commands#xx1268462) for additional details. 
```

File: Packs/AristaSwitch/README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+- By default, the timestamps in the generated event logs are specified in the traditional [RFC3164](https://www.ietf.org/rfc/rfc3164.txt) syslog format, which does not include a year and a timezone. It is recommended you override this default setting and configure the switch to forward the Syslog messages in [RFC5424](https://datatracker.ietf.org/doc/html/rfc5424) format, which specifies a high-resolution [RFC3339](https://datatracker.ietf.org/doc/html/rfc3339) timestamp which does include a year and a timezone. This configuration could be done from the _Global Configuration_ command mode via the "_**logging format**_" command. See [Syslog Logging Format](https://www.arista.com/en/um-eos/eos-switch-administration-commands#xx1268462) for additional details. 
+- This configuration guide was brief and basic. For the full documentation, be sure to check the latest Arista Configuration Guide for your switch version. In addition, you may find the following links useful:
```
Comment (ShirleyDenkberg) [inline Packs/AristaSwitch/README.md:38]:
```suggestion
- The configuration described above was brief and basic. For the full documentation, be sure to see the latest Arista Configuration Guide for your switch version. In addition, you may find the following links useful: 
```

**Comment by ShirleyDenkberg** [comment]
@cweltPA Doc review completed.
FYI, I am not a reviewer so I cannot approve this PR.

**Comment by cweltPA** [comment]
@DeanArbel 

Hey, 

Seems that this PR validations fail only due to Modeling Rules Test-Data mismtaches. 
These Test-Data mismatches are false-positives: 

They occur because the modeling rules use a _[RULE]_ stanza section, 
which is currently unsupported by the Test-Data SDK functinality (The XQL query that is generated by the SDK _test_ does not take into account the XDM fields that are mapped by invoking modeling rules that are defined under the _[RULE]_ sections,   so as a result, the " _| fields <Field1>, <Field2>, ... _ " stage on the XQL query does include all fields, and for these fields a NULL value is returned and fails the Test-Data validation. 

There is a dev-task issue for this matter:
[modeling-rules lack support in [RULE] sections](https://jira-hq.paloaltonetworks.local/browse/CIAC-6590)

Hence, requesting a force merge for this PR. 

Chanan 

@evisochek FYI

**Review by evisochek — APPROVED** [review]

**Pull request merged**